packages/circuits/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@zk-kit/circuits",
-    "version": "0.2.4",
+    "version": "0.2.5",
```

Comment:
@cedoor are you okay with this bump?

Reply:
Umh I think it should be at least a minor in this case, as there are additional circuits, not patches.

Reply:
0.3.0 then?

Reply:
Yes, sounds great!

Reply:
Done